pkg/api/v1/client/proxy.go

```diff
@@ -443,12 +443,12 @@ func (c ProxyScriptsAPI) DownloadFile(executionID, fileName, destination string)
 		Suffix(uri).
 		SetHeader("Accept", "text/event-stream").
 		Stream(context.Background())
```

Comment:
@exu how do you check for status code in such calls?

Reply:
@jasmingacic look into makeDeleteRequest, something like that
`		var code int
		resp.StatusCode(&code)
		if code != http.StatusNoContent {
			respBody, err := resp.Raw()
			if err != nil {
				return err
			}
			return fmt.Errorf("request returned error: %s", respBody)
		}`

Reply:
I would if i had any response here.
Let's merge it and then we deal with it later